### PR TITLE
Support WalletConnect in mobile

### PR DESCRIPTION
**Pull request by nwingt**

### CODE REVIEW

File: util/WalletConnect.js [hunk 1]
```diff
@@ -119,6 +127,23 @@ class WalletConnect {
     }
   }
 
+  async requestForLogin(loginMessage) {
+    const payload = await Promise.race([
+      this.connector.sendCustomRequest({
+        id: payloadId(),
+        jsonrpc: '2.0',
+        method: 'likerId_login',
```
Comment (williamchong) [inline util/WalletConnect.js:135]: Is this a special message for iOS to do both init and sign login?

File: .circleci/config.yml [hunk 2]
```diff
@@ -293,6 +292,53 @@ jobs:
...
+            docker tag like-store:latest us.gcr.io/${PROJECT_NAME}/like-store-frontend:legacy-${CIRCLE_SHA1}
+      - run:
+          name: Push images
+          command: |
+            gcloud docker -- push us.gcr.io/${PROJECT_NAME}/like-store-frontend:latest
+            gcloud docker -- push us.gcr.io/${PROJECT_NAME}/like-store-frontend:legacy-${CIRCLE_SHA1}
+      - deploy:
+          name: Deploy
+          command: |
+            kubectl patch deployment like-store-frontend-legacy -p '{"spec":{"template":{"spec":{"containers":[{"name":"like-store-frontend","image":"us.gcr.io/'"$PROJECT_NAME"'/like-store-frontend:legacy-'"$CIRCLE_SHA1"'"}]}}}}'
```
Comment (williamchong) [inline .circleci/config.yml:341]: if `legacy` doesn't set any extra build/env var, just the SHA hash would be good?

File: package.json [hunk 3]
```diff
@@ -72,6 +73,7 @@
     "gsap": "^3.0.4",
     "helmet": "^3.21.2",
     "http-proxy-middleware": "^2.0.1",
+    "js-cookie": "^3.0.1",
```
Comment (williamchong) [inline package.json:76]: use tiny-cookie too?

**Review by williamchong — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 2 above)
Comment (nwingt) [reply .circleci/config.yml:341]: Do you mean keeping `main` instead of `legacy`?

Comment (williamchong) [reply .circleci/config.yml:341]: I mean we don't need separated tag for `main` and `legacy`, the commit hash should be enough
on the other hand if we set `IS_TESTNET` or other env in build time, we would need a separated tag

Comment (nwingt) [reply .circleci/config.yml:341]: Do you want to keep `main` in the tag for mainnet/legacy image?

Comment (williamchong) [reply .circleci/config.yml:341]: yes please keep `main` vs `test`/`dev`

**Review by williamchong — APPROVED** [review]

**Pull request merged**